src/modules/flow/http-client/http-client.c

```diff
@@ -1814,9 +1820,10 @@ create_url_create_process(struct sol_flow_node *node, void *data,
 
     r = sol_http_create_uri(&uri, url, &mdata->params);
     SOL_INT_CHECK(r, < 0, r);
-    r = sol_flow_send_string_packet(node,
-        SOL_FLOW_NODE_TYPE_HTTP_CLIENT_CREATE_URL__OUT__OUT, uri);
-    free(uri);
+    r = sol_flow_send_string_slice_packet(node,
+        SOL_FLOW_NODE_TYPE_HTTP_CLIENT_CREATE_URL__OUT__OUT,
+        sol_buffer_get_slice(&uri));
+    sol_buffer_fini(&uri);
```

Comment:
Sounds fair, there are other parts of this PR that I will sol_flow_send_string_slice_packet(), I will change them.